app/services/proofing/lexis_nexis/date_formatter.rb

```diff
@@ -24,8 +24,8 @@ def yyyymmdd
       def parse_date_string(date_string)
         if /\A\d{8}\z/.match?(date_string)
           Date.strptime(date_string, '%Y%m%d')
-        elsif %r{\A\d{2}/\d{2}/\d{4}\z}.match?(date_string)
-          Date.strptime(date_string, '%m/%d/%Y')
+        else
+          DateParser.parse_legacy(date_string)
```

Comment:
Am I following the issue correctly: We started storing `applicant[:dob]` in the new YYYY-MM-DD format (previously MM/DD/YYYY), so this method was returning `nil`?

`parse_legacy` seems to handle it regardless, but could we have any confidence here that the date will be `YYYY-MM-DD` enough to just... ?

```suggestion
          Date.parse(date_string)
```

Reply:
I think that is a correct understanding... at this point we shouldn't be seeing the YYYYMMDD (no punctuation form) either so maybe we can remove that? how bold are you feeling?

Reply:
> how bold are you feeling?

😆  After today, not very. I'm fine with keeping it as-is.

Trying to find _where exactly_ we set this `dob`, I'm also finding some specs that have the old format. Do we want to update these?

Example:

https://github.com/18F/identity-idp/blob/b73a530329ad5fea196e7e2aa25675df1a6b779a/spec/services/proofing/lexis_nexis/phone_finder/proofing_spec.rb#L11

Reply:
My mental model of this process is:
1. image of license uploaded to vendor (code lives in identity-doc-auth)
2. returned payload has `dob` key
3. we pass that payload here

so since that gem is updated to only send YYYY-MM-DD, we should be able to use `Date.parse`

However  because of the ~15min deploy there may be some old data in there) so we may want to do an approach similar to https://github.com/18F/identity-idp/pull/5191/files#diff-c4661faa79ade4098b2ffdd720d798c212e2fd197ff60e1ddae26c73a64dd830R51-R53 and use `DateParser.parse_legacy` for now

Reply:
Turns out the `Date.parse` accepts the YYYYMMDD format (no puctuation) natively, and `#yyyymmdd` appeared unused so removed that entirely  in fb546cf5d1dfad370d2adb20c16cb460a66d2d81